source/cl/source/extension/include/extension/extension.h

```diff
@@ -649,6 +681,28 @@ cl_int GetKernelArgInfo(cl_kernel kernel, cl_uint arg_indx,
                         cl_kernel_arg_info param_name, size_t param_value_size,
                         void *param_value, size_t *param_value_size_ret);
 
+#if defined(CL_VERSION_3_0)
+/// @brief Aggregate all kernel sub group information.
+///
+/// @param kernel Kernel to query.
+/// @param device Device being targeted.
+/// @param param_name Parameter name to query.
+/// @param input_value_size Size in bytes of memory pointed to by
+/// input_value.
+/// @param input_value Pointer to memory where parameterization of query is
+/// passed from.
+/// @param param_value_size Size of `param_value` storage.
```

Comment:
I always think it's handy to say the units e.g. in number of bytes.

Reply:
Agreed, though I was just copying the doxygen style from the other extension APIs in this file - none of which mention 'bytes'.

Should I change all of them to say 'bytes'? Might start to veer into "unrelated changes" territory.

Reply:
fair enough, I think it's reasonably clear.